janus_core/calculations/md.py

```diff
@@ -218,6 +231,8 @@ def __init__(
         post_process_kwargs: PostProcessKwargs | None = None,
         correlation_kwargs: list[CorrelationKwargs] | None = None,
         seed: int | None = None,
+        enable_progress_bar: bool = False,
+        progress_bar_update_every: int = 1,
```

Comment:
We may want to consider a higher default, assuming most realistic use cases will be doing thousands - millions of steps?